modules/editable-layers/src/editable-layers/editable-layer.ts

```diff
@@ -15,7 +15,7 @@ import {
 } from '../edit-modes/types';
 import {Position} from '../utils/geojson-types';
 
-const EVENT_TYPES = ['anyclick', 'pointermove', 'panstart', 'panmove', 'panend', 'keyup'];
```

Comment:
@Pessimistress is this expected?

Reply:
mjolnir v3 no longer registers a default set of gesture recognizers. The recognizers used by deck are defined here:

https://github.com/visgl/deck.gl/blob/623265907fe2db31a91c9d2965e73c73cbf7efc1/modules/core/src/lib/constants.ts#L96-L102